change/@passageidentity-passage-node-5524b98c-be4e-4e30-bb89-94cadf77ecb7.json

```diff
@@ -0,0 +1,7 @@
+{
+    "type": "major",
```

Comment:
any particular reason this is a major change?

Reply:
It was done to indicate a breaking change, since 2 fields are removed. If there's a consumer using the fields, they'd need to take some action to update.

& the [latest tag](https://github.com/passageidentity/passage-node/tags) in this repo is v1.11.1 so I was thinking it'd need a major bump.